raiden/utils/cli.py

```diff
@@ -268,3 +273,4 @@ def convert(self, value, param, ctx):
 
 ADDRESS_TYPE = AddressType()
 LOG_LEVEL_CONFIG_TYPE = LogLevelConfigType()
+DEFAULT_OPTION = {}
```

Comment:
Please don't store the defaults in a global. If there are identically named parameters they will conflict.

Also, you don't need to store the defaults separately at all since you can access the default values of the defined options of a command (in this case `run`) like this:
```
optionname_to_default = {param.name: param.get_default(ctx) for param in run.params}
```

Reply:
i didn't know about that ^^